tests/test_interpolation.py

```diff
@@ -281,4 +365,4 @@ def test_position(shape):
 
 
 if __name__ == "__main__":
-    test_interpolate_custom((11, 11), [(.05, .9), (.01, .8)])
+    test_inject_time_shift((11, 11), [(.05, .95), (.45, .45)], 1.)
```

Comment:
unnecessary?

Reply:
I like the `__main__` :) but can remove it

Reply:
Oh yeah, usually set to the last failing one :D I'll remove it